Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.24
```

Comment:
I'm guessing we might want to multistage this so the actual binary is running on like `FROM scratch`

Reply:
We can do this later tho